lib/fog/ovirt/requests/compute/v4/create_vm.rb

```diff
@@ -59,6 +59,8 @@ def create_vm(attrs)
 
             attrs[:memory_policy] = OvirtSDK4::MemoryPolicy.new(:guaranteed => attrs[:memory]) if attrs[:memory].to_i < Fog::Compute::Ovirt::DISK_SIZE_TO_GB
 
+            attrs[:high_availability] = OvirtSDK4::HighAvailability.new(:enabled => attrs[:ha])
```

Comment:
What happens if `attrs` does not contain `:ha`?

Reply:
attrs will contain :ha in case checkbox is off and on.